scrapy/templates/spiders/csvfeed.tmpl

```diff
@@ -2,14 +2,14 @@ from scrapy.spiders import CSVFeedSpider
 
 
 class $classname(CSVFeedSpider):
-    name = '$name'
-    allowed_domains = ['$domain']
-    start_urls = ['http://$domain/feed.csv']
-    # headers = ['id', 'name', 'description', 'image_link']
-    # delimiter = '\t'
+    name = "$name"
+    allowed_domains = ["$domain"]
+    start_urls = ["http://$domain/feed.csv"]
+    #headers = ['id', 'name', 'description', 'image_link']
```

Comment:
Wow, things are more complicated that I foresaw… We should probably have the commented out code in these files also *blacked* somehow.

Reply:
Okay  on it